sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -274,8 +274,8 @@ def _open(self):
             self._close_handler()
             raise
 
-    def _send(self, message, timeout=None):
-        # type: (Union[ServiceBusMessage, ServiceBusMessageBatch], Optional[float]) -> None
+    def _send(self, message, timeout=None, last_exception=None):
```

Comment:
had removed last_exception previously --- it still gets passed through via one of the tests somehow, not used in pyamqp but leaving here for uamqp switch purposes ***